docker/nugus_sim/run.py

```diff
@@ -7,32 +7,40 @@
 
 import ruamel.yaml
 
-BINARIES_DIR = "/home/nubots/NUbots/binaries"
-CONFIG_DIR = BINARIES_DIR + "/config"
+DEFAULT_BINARIES_DIR = "/home/nubots/NUbots/binaries"
+DEFAULT_CONFIG_DIR = DEFAULT_BINARIES_DIR + "/config"
```

Comment:
Sorry, I should have picked up on this earlier, but we should be consistent with path usages and should always use `os.path.join` with constructing paths
```suggestion
DEFAULT_BINARIES_DIR = os.path.abspath(os.path.join(os.sep, "home", "nubots", "NUbots", "binaries"))
DEFAULT_CONFIG_DIR = os.path.join(DEFAULT_BINARIES_DIR, "config")
```